client/analytics/report/products/config.js

```diff
@@ -26,7 +28,9 @@ export function getCategoryLabelsById( queryString ) {
 		include: idList.join( ',' ),
 		per_page: idList.length,
 	} );
-	return apiFetch( { path: `${ NAMESPACE }products/categories${ payload }` } );
+	return apiFetch( { path: `${ NAMESPACE }products/categories${ payload }` } ).then( data =>
+		data.map( product => ( { id: product.id, label: product.name } ) )
```

Comment:
Should the mapping be categories into category id and labels, not product ones?

Reply:
I mean, on further reflection this does work - but the map currentValue argument should be `category` and not `product` methinks